makefile

```diff
@@ -188,6 +188,17 @@ dexrt-llvm: src/lib/dexrt.bc
 %.bc: %.cpp
 	$(CLANG) $(CXXFLAGS) -DDEX_LIVE -c -emit-llvm $^ -o $@
 
+# --- downloading tutorial data ---
+
+examples/:
+	mkdir -p $@
+
+examples/%.gz: | examples/
+	wget -qO $@ http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/$(@F)
```

Comment:
That feels like a bit too much automation, don't you think? It's not like every gz file can be obtained from that URL. Is there a nice way to specify this as the recipe for the two files we're using at the moment?